src/components/badge.component.js

```diff
@@ -10,39 +10,29 @@ type Props = {
   largeText: boolean,
 };
 
-const styles = StyleSheet.create({
-  badge: {
-    flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center',
-    borderRadius: 18,
-    width: 18,
-    height: 18,
-    borderColor: colors.alabaster,
-    borderWidth: 1,
-  },
-  badgeText: {
-    ...fonts.fontPrimaryBold,
-    backgroundColor: 'transparent',
-  },
-  textSmall: {
-    fontSize: normalize(7),
-  },
-  textLarge: {
-    fontSize: normalize(9.5),
-  },
-});
+const BadgeContainer = styled.View`
+  flex: 1;
+  align-items: 'center';
+  justify-content: 'center';
+  border-radius: 18;
+  width: 18;
+  height: 18;
+  border-color: ${colors.alabaster};
+  border-width: 1;
+  ${({ backgroundColor }) => `background-color: ${backgroundColor};`};
+`;
+
+const BadgeText = styled.Text`
+  ${{ ...fonts.fontPrimaryBold }};
+  background-color: 'transparent';
+  ${({ largeText }) =>
```

Comment:
Same goes for this `font-size: ${largeText ? normalize(9.5) : normalize(7)}`

Reply:
Hi @alejandronanez , might not work since unlike `border-color: ${colors.alabaster};`, `colors` is a file-wide variable whereas `backgroundColor` and `largeText` are props passed down to the styled components. https://www.styled-components.com/docs/basics#adapting-based-on-props

Reply:
@alejandronanez , let me know if I can help you with anything else

Reply:
Oh yeah, you're right. I missed that, sorry.
Thanks!

Reply:
the usual way would be this though: 
```
...
    font-size: ${({largeText}) => largeText ? normalize(9.5) : normalize(7)};
```

Reply:
I agree that would be more concise. I guess I just made the code style more consistent by making it parallel to how I wrote BadgeContainer's `backgroundColor` which doesn't have a default value. Since writing something without a default value that way would make it `background-color: undefined` which, without tree-shaking, would yield dead code.